lib/huginn_agent/templates/newagent/spec/spec_helper.rb.tt

```diff
@@ -0,0 +1,7 @@
+base_path = File.expand_path('../../', __FILE__)
```

Comment:
I see, so having `spec_helper.rb` inside the gem is needed if you want to use files in `spec/support` and run the specs with `rspec` directly?